client/index.html

```diff
@@ -2,10 +2,16 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <link rel="stylesheet" href="./index.css">
-  <script type="module" src="./index.js" defer></script>
   <meta name="color-scheme" content="dark light">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="A page to display compatible browsers from a browserslist string">
+  <!-- TODO Add og.png (1200x630) <meta property="og:image" content="/og.png"> -->
+  <link rel="icon" href="/favicon.ico" sizes="any">
+  <link rel="icon" href="/assets/icon.svg" type="image/svg+xml">
+  <link rel="apple-touch-icon" href="/assets/apple-touch-icon.png">
```

Comment:
I think we need `./` path to enable cache buster? Maybe I am wrong, can you show HTML after build?

Reply:
![image](https://user-images.githubusercontent.com/22644149/183536098-fddc7f88-8b19-45df-be0d-1f36934c37dc.png)

Oh, even worse than I expected

We can prevent it:
https://vitejs.dev/config/build-options.html#build-assetsinlinelimit

Reply:
[assetsinlinelimit](https://vitejs.dev/config/build-options.html#build-assetsinlinelimit)

seems to be sometimes this useful behavior

too bad it can't be used optionally :(

Reply:
Don’t worry, for modern stack it is better to disable `data:uri` and use `prelaod` instead. Glory to HTTP/2.

The problem of `data:uri` is parsers. To parse PNG inside HTML, you first need to run HTML parser, than base64 parser and only then PNG parser. With HTTP/2 and preload you will use less CPU and got result even faster.